nevergrad/parametrization/discretization.py

```diff
@@ -42,9 +42,13 @@ def threshold_discretization(x: tp.ArrayLike, arity: int = 2) -> tp.List[int]:
 # The function below is the opposite of the function above.
 def inverse_threshold_discretization(indexes: tp.List[int], arity: int = 2) -> np.ndarray:
     indexes_arr = np.array(indexes, copy=True)
+    assert not np.any(np.isnan(indexes_arr))
     pdf_bin_size = 1 / arity
     # We take the center of each bin (in the pdf space)
-    return scipy.stats.norm.ppf(indexes_arr * pdf_bin_size + (pdf_bin_size / 2))  # type: ignore
+    x = scipy.stats.norm.ppf(indexes_arr * pdf_bin_size + (pdf_bin_size / 2))  # type: ignore
+    nan_indices = np.where(np.isnan(x))
+    x[nan_indices] = np.sign(indexes_arr[nan_indices] - (arity / 2.)) * 500.
+    return x
```

Comment:
are those changes still needed if you are using 500 somewhere else? having such constants in 2 different place is dangereous.

Reply:
There is no reason for choosing 500 in both places. Actually in both places it should be something better.
In the other place, this should depend on other quantities in the problem.
Here, this is just something like a max float. Let me improve this.